packages/phone/src/Phone.js

```diff
@@ -43,16 +43,23 @@ const Phone = ({ name, label, country = 'US', showExtension = false, extProps, p
 };
 
 Phone.propTypes = {
+  /** Identifies the field and matches the validation schema. */
   name: PropTypes.string.isRequired,
-  label: PropTypes.string,
-  showExtension: PropTypes.bool,
-  country: PropTypes.string,
-  phoneColProps: PropTypes.object,
+  /** Used to pass props to the extension field when it is enabled. */
   extProps: PropTypes.shape({
     name: PropTypes.string.isRequired,
     label: PropTypes.string,
+    /** Used to control props on <Col /> for the extension field, if needed. The extension column has no default size value, so it's default will effectively be size: { 12 - phoneColSize } unless otherwise specified. */
     extColProps: PropTypes.object,
   }),
+  /** Displays a Reactstrap <Label /> for the field. */
```

Comment:
When I run locally it looks like `<Label />` isn't escaped. I'd surround it with backtics. This should be the last thing.